src/main/java/xyz/vegaone/easytrackingv2/service/ProjectService.java

```diff
@@ -45,20 +77,42 @@ public Project createProject(Project project) {
 
     }
 
+    @Transactional
     public Project getProject(Long id) {
-        ProjectEntity projectEntity = projectRepo.findOne(id);
 
-        if (projectEntity == null) {
-            throw new EntityNotFoundException("Project with id " + id + " not found");
-        }
+        Optional<ProjectEntity> projectOptional = projectRepo.findById(id);
+
+        if (projectOptional.isPresent()) {
```

Comment:
Instead of having the if statement, we should take advantage of the optional methods. For example, the if can be replaced with: 
`ProjectEntity projectEntity = projectOptional.orElseThrow(() -> new EntityNotFoundException("Project with id " + id + " not found"));` 
which will either return the value inside optional (if present) or throw an exception.